viewer/packages/camera-manager/src/types.ts

```diff
@@ -106,3 +107,14 @@ export const CAMERA_MANAGER_EVENT_TYPE_LIST = ['cameraChange', 'cameraStop'] as
  * Union type of the supported camera manager event types
  */
 export type CameraManagerEventType = (typeof CAMERA_MANAGER_EVENT_TYPE_LIST)[number];
+
+export type NearAndFarPlaneBuffers = {
+  corners: Vector3[];
+  cameraPosition: Vector3;
+  cameraDirection: Vector3;
+};
```

Comment:
I think this type and the one below can be moved into `CameraManagerHelper.ts`, since they're only being used there